projects/systelab-components/src/lib/contextmenu/context-menu-submenu-item.component.html

```diff
@@ -0,0 +1,23 @@
+<li *ngFor="let childAction of action.childrenContextMenuOptions"
+    class="d-flex align-items-center" [class.disabled]="!true"
+    (click)="doClickWithString($event, parentId, actionName + levelSeparator + childAction.actionId)"
+    (mouseover)="doMouseOver($event, elementID, actionName + levelSeparator + childAction.actionId)">
+    <div *ngIf="isEnabled" class="slab-flex-1" style="flex:1 1 auto">
```

Comment:
flex:1 1 auto. ?    slab-flex-1 is already flex:1

Reply:
Sorry I'm not an expert on styles.